scripts/lint_prowjobs.go

```diff
@@ -0,0 +1,279 @@
+		postsubmitErrors[postsubmitFileName] = append(postsubmitErrors[postsubmitFileName], errorString)
+	}
+
+	bucketLineNo, _ := findLineNumber(fileContentsString, "bucket:")
+	jobBucket := postsubmitType.JobBase.UtilityConfig.DecorationConfig.GCSConfiguration.Bucket
+	if jobBucket != postsubmitConstants.Bucket {
+		errorString := bucketLineNo + "\tIncorrect bucket configuration, please configure S3 bucket as => bucket: " + postsubmitConstants.Bucket
+		postsubmitErrors[postsubmitFileName] = append(postsubmitErrors[postsubmitFileName], errorString)
+	}
+
+	serviceAccountLineNo, _ := findLineNumber(fileContentsString, "serviceaccountName:")
+	jobServiceAccountName := postsubmitType.JobBase.Spec.ServiceAccountName
+	if (strings.Contains(postsubmitFileName, "helm-chart") || strings.Contains(postsubmitFileName, "athens")) && jobServiceAccountName != postsubmitConstants.AthensHelmServiceAccountName {
+		errorString := serviceAccountLineNo + "\tIncorrect service account configuration, please configure service account as => serviceaccountName: " + postsubmitConstants.AthensHelmServiceAccountName
+		postsubmitErrors[postsubmitFileName] = append(postsubmitErrors[postsubmitFileName], errorString)
+	} else if strings.Contains(postsubmitFileName, "docs") && jobServiceAccountName != postsubmitConstants.DocsServiceAccountName {
+		errorString := serviceAccountLineNo + "\tIncorrect service account configuration, please configure service account as => serviceaccountName: " + postsubmitConstants.DocsServiceAccountName
+		postsubmitErrors[postsubmitFileName] = append(postsubmitErrors[postsubmitFileName], errorString)
+	} else if jobServiceAccountName != postsubmitConstants.DefaultServiceAccountName {
+		errorString := serviceAccountLineNo + "\tIncorrect service account configuration, please configure service account as => serviceaccountName: " + postsubmitConstants.DefaultServiceAccountName
+		postsubmitErrors[postsubmitFileName] = append(postsubmitErrors[postsubmitFileName], errorString)
+	}
+
+	makeCommandLineNo, _ := findLineNumber(fileContentsString, "make")
+	jobMakeTargetMatches := regexp.MustCompile(`make (\w+) .*`).FindStringSubmatch(strings.Join(postsubmitType.JobBase.Spec.Containers[0].Command, " "))
+	jobMakeTarget := jobMakeTargetMatches[len(jobMakeTargetMatches)-1]
+	if jobMakeTarget != postsubmitConstants.MakeTarget {
+		errorString := makeCommandLineNo + "\tInvalid make target, please use the \"" + postsubmitConstants.MakeTarget + "\" target."
+		postsubmitErrors[postsubmitFileName] = append(postsubmitErrors[postsubmitFileName], errorString)
+	}
+}
+
+func findLineNumber(fileContentsString string, searchString string) (string, error) {
+	line := 1
+
+	fileLines := strings.Split(fileContentsString, "\n")
+
+	for _, fileLine := range fileLines {
+		if strings.Contains(fileLine, searchString) {
+			return strconv.Itoa(line), nil
+		}
+		line++
+	}
+
+	return "0", nil
+}
```

Comment:
You never returned an error, so do you really need that in the response?
```suggestion
func findLineNumber(fileContentsString string, searchString string) int {
	fileLines := strings.Split(fileContentsString, "\n")
	for lineNo, fileLine := range fileLines {
		if strings.Contains(fileLine, searchString) {
			return lineNo + 1
		}
	}
	return 0
}
```